tests/test_src/test_wrappers/test_wrappers.cpp

```diff
@@ -149,3 +150,50 @@ TEST(CryptoWrapper, getSignature)
     EXPECT_EQ(util::stringToHex(signature), expectedSignature);
 
 }
+
+TEST(ClientWrapper, get_validClient)
+{
+    wrapper::http::Client client("https://api.elrond.com");
```

Comment:
Perhaps use `https://gateway.elrond.com` instead? Or even better, `https://testnet-gateway.elrond.com` - so that we don't hit the Mainnet API when running the tests?

Reply:
Changed it to `https://testnet-gateway.elrond.com`. I was not aware of them, I followed the documentation from here: https://docs.elrond.com/sdk-and-tools/rest-api/rest-api/